### PR TITLE
Car port:  CADILLAC ESCALADE ESV PLATINUM 2019

### PR DESCRIPTION
**Pull request by twilsonco**
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py ~~and ran `selfdrive/car/docs.py` to generate new docs~~\*
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot: 168f8b3be57f66ae|2023-09-12--21-44-42
- [x] route with stock system: 168f8b3be57f66ae|2023-09-13--17-22-33

\* cloned OP to linux machine, ran ubuntu_setup.sh, enter `pipenv shell`, installed missing python dependencies, ran `scons` and build successfully, then ran `selfdrive/car/docs.py` which throws: 
```
(openpilot) haiiro@haiiro-pc:~/optw/openpilot/openpilot$ ./selfdrive/car/docs.py
Traceback (most recent call last):
  File "./selfdrive/car/docs.py", line 14, in <module>
    from openpilot.selfdrive.car.car_helpers import interfaces, get_interface_attr
  File "/home/haiiro/optw/openpilot/openpilot/selfdrive/car/car_helpers.py", line 6, in <module>
    from openpilot.common.params import Params
  File "/home/haiiro/optw/openpilot/openpilot/common/params.py", line 1, in <module>
    from openpilot.common.params_pyx import Params, ParamKeyType, UnknownKeyName, put_nonblocking, \
ImportError: /home/haiiro/optw/openpilot/openpilot/common/params_pyx.so: undefined symbol: PyGC_Disable
```
I figure I'm just missing something here, but I tried!

### CODE REVIEW

File: selfdrive/car/gm/interface.py [hunk 1]
```diff
@@ -247,6 +247,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
...
 
+    elif candidate == CAR.ESCALADE_ESV_2019:
+      ret.minEnableSpeed = -1.  # engage speed is decided by pcm
+      ret.mass = 2739.
+      ret.wheelbase = 3.302
+      ret.steerRatio = 17.3
+      ret.centerToFront = ret.wheelbase * 0.5
+      ret.steerActuatorDelay = 0.2
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.tire_stiffness_factor = 1.0
```
Comment (sshane) [inline selfdrive/car/gm/interface.py:258]:
```suggestion
      ret.tireStiffnessFactor = 1.0
```

Comment (twilsonco) [reply selfdrive/car/gm/interface.py:258]: oops. so many refactors happened since I first drafted this branch 😅

**Review by twilsonco — COMMENTED** [review]
approved

File: selfdrive/car/gm/interface.py [hunk 2]
```diff
@@ -247,6 +247,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
...
 
+    elif candidate == CAR.ESCALADE_ESV_2019:
+      ret.minEnableSpeed = -1.  # engage speed is decided by pcm
+      ret.mass = 2739.
+      ret.wheelbase = 3.302
+      ret.steerRatio = 17.3
+      ret.centerToFront = ret.wheelbase * 0.5
+      ret.steerActuatorDelay = 0.2
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.tireStiffnessFactor = 1.0
```
Comment (sshane) [inline selfdrive/car/gm/interface.py:258]: Can we merge with `ESCALADE_ESV`? No need for a new platform if we can help it.

**Review by sshane — CHANGES_REQUESTED** [review]

File: selfdrive/car/gm/interface.py (hunk 2 above)
Comment (twilsonco) [reply selfdrive/car/gm/interface.py:258]: My only concern there is that we ended up with lat accel factor 1.3 based on what felt best, vs 1.9 for the other Escalade. That 50% difference will matter.

**Comment by qeggs** [comment]
I drove two additional road tests comparing the prior version of the PR branch with the current version. 

Here is a route using the prior version of the PR branch:
168f8b3be57f66ae|2023-09-23--14-43-46--0

Here is the same route using the current version of the PR branch:
168f8b3be57f66ae|2023-09-23--18-27-23--0

**Comment by sshane** [comment]
Will check it out in a bit, but which do you prefer?

**Comment by qeggs** [comment]
I think there is a noticeable improvement with the newer version.

File: selfdrive/car/torque_data/override.yaml [hunk 3]
```diff
@@ -37,6 +37,7 @@ RAM 1500 5TH GEN: [2.0, 2.0, 0.05]
 RAM HD 5TH GEN: [1.4, 1.4, 0.05]
 SUBARU OUTBACK 6TH GEN: [2.0, 2.0, 0.2]
 CADILLAC ESCALADE 2017: [1.899999976158142, 1.842270016670227, 0.1120000034570694]
+CADILLAC ESCALADE ESV 2019: [1.15, 1.5, 0.2]
```
Comment (sshane) [inline selfdrive/car/torque_data/override.yaml:40]:
```suggestion
CADILLAC ESCALADE ESV 2019: [1.15, 1.3, 0.2]
```

File: RELEASES.md [hunk 4]
```diff
@@ -2,6 +2,7 @@ Version 0.9.5 (2023-XX-XX)
 ========================
 * New driving model
   * Improved navigate on openpilot performance using navigation instructions as an additional model input
+* Cadillac Escalade ESV 2019 support thanks to twilsonco!
```
Comment (sshane) [inline RELEASES.md:5]:
```suggestion
* Cadillac Escalade ESV 2019 support thanks to twilsonco!
```

**Comment by sshane** [comment]
thanks!

File: selfdrive/car/gm/interface.py
Comment (twilsonco) [inline selfdrive/car/gm/interface.py:1]: @sshane I'm confused. I PR'd it with the torque controller, and then you made adjustments to the lat accel factor  based on logs and it drove even better after that, so why is it back to the latPID controller?

**Pull request merged**